homeworks/roman.lytvynenko_Lytvynenko18/7-arrays-loops/arrays.js

```diff
@@ -0,0 +1,78 @@
+const arrayOne = [3, 0, -5, 1, 44, -12, 3, 0, 0, 1, 2, -3, -3, 2, 1, 4, -2, -3, -1];
+const arrayTwo = [-1, -8, -2];
+const arrayThree = [1, 7, 3];
+const arrayFour = [1, undefined, 3, 5, -3];
+const arrayFive = [1, NaN, 3, 5, -3];
+
+function maxNumberOfArr(array) {
+    let maxNumber = array[0];
+    for (let i = 1; i < array.length; i++) {
+        if (array[i] > maxNumber) {
+            maxNumber = array[i];
+        }
+    }
+    return maxNumber;
+    // write your code here and instead of array return maxNumber
+}
+
+console.log(maxNumberOfArr(arrayOne)); // 44
+console.log(maxNumberOfArr(arrayTwo)); // -1
+console.log(maxNumberOfArr(arrayThree)); // 7
+console.log(maxNumberOfArr(arrayFour)); // 5
+console.log(maxNumberOfArr(arrayFive)); // 5
+
+function minNumberOfArr(array) {
+    let minNumber = array[0];
+    for (let i = 1; i < array.length; i++) {
+        if (array[i] < minNumber) {
+            minNumber = array[i];
+        }
+    }
+    return minNumber;
+    // write your code here and instead of array return minNumber
+}
+
+console.log(minNumberOfArr(arrayOne)); // -12
+console.log(minNumberOfArr(arrayTwo)); // -8
+console.log(minNumberOfArr(arrayThree)); // 1
+console.log(minNumberOfArr(arrayFour)); // -3
+console.log(minNumberOfArr(arrayFive)); // -3
+
+function sumNumbersOfArr(array) {
+    let sumNumbers = 0;
+    for (let i = 0; i < array.length; i++) {
+        if (!Number.isNaN(array[i])) {
+            sumNumbers += array[i];
+        }
+    }
+    return sumNumbers;
+    // write your code here and instead of array return sum of numbers
+}
+
+console.log(sumNumbersOfArr(arrayOne)); // 32
+console.log(sumNumbersOfArr(arrayTwo)); // -11
+console.log(sumNumbersOfArr(arrayThree)); // 11
+console.log(sumNumbersOfArr(arrayFour)); // 6
```

Comment:
Here your function returns `NaN`

Reply:
Thank's for review.
Improved check in aa0da52c7c07b7906c577a01d1c96d079ff6062d

function sumNumbersOfArr(array) {
    let sumNumbers = 0;
    for (let i = 0; i < array.length; i++) {
        **if (typeof array[i] === 'number' && !Number.isNaN(array[i])) {**
            sumNumbers += array[i];
        }
    }
    return sumNumbers;